ftplugin/latex-suite/compiler.vim

```diff
@@ -591,7 +591,7 @@ function! Tex_CompileMultipleTimes()
 		" The first time we see if we need to generate the bibliography and if the .bbl file
 		" changes, we will rerun latex.
 		" We use '\\bibdata' as a check for BibTeX and '\\abx' as a check for biber.
-		if runCount == 0 && Tex_IsPresentInFile('\v\\bibdata|\\abx', mainFileName_root.'.aux')
+		if runCount == 0 && Tex_IsPresentInFile('\\bibdata|\\abx', mainFileName_root.'.aux')
```

Comment:
Because the bibtex cannot run automatically with the '\v' on my computer. It works well without it.
I think probably because I am working on windows and there is no vertical tab before the \bibdata line.